src/internal_events/http_client.rs

```diff
@@ -109,3 +109,16 @@ impl<'a, B: HttpBody> std::fmt::Display for FormatBody<'a, B> {
         }
     }
 }
+
+#[derive(Debug)]
+pub struct HttpBodyDataBuffer {
```

Comment:
See [the event naming docs](https://github.com/vectordotdev/vector/blob/master/docs/specs/instrumentation.md#event-naming) for the standards on how these are to be named.